src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String COMMA = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] fileData = readFile(fromFileName).split(NEW_LINE);
+        String[] resultData = prepareStingToWrite(fileData);
+        clearFile(toFileName);
```

Comment:
If you intend to overwrite the file content, consider using new FileWriter(file, false) to truncate the file before writing.